graph-builder/src/main.rs

```diff
@@ -54,8 +54,7 @@ fn main() -> Result<(), Error> {
                 2 => LevelFilter::Debug,
                 _ => LevelFilter::Trace,
             },
-        )
```

Comment:
Don't accept the `rustfmt` suggestions from stable. It suffers from a [formatting bug](https://github.com/rust-lang-nursery/rustfmt/issues/2932).

Reply:
Okay, odd enough ;-) I'll remember this when setting up the CI checks.